e2e/integrations/multiple-steps-integrations.feature

```diff
@@ -74,3 +74,91 @@ Feature: Test to verify advanced integration with multiple steps
 
     When Camilla deletes the "Two steps integration" integration
     Then Camilla can not see "Two steps integration" integration anymore
+    
+  Scenario: Deleting steps on integration    
+    When "Camilla" navigates to the "Home" page
+    And clicks on the "Create Integration" button to create a new integration.
+    Then she is presented with a visual integration editor
+    And she is prompted to select a "Start" connection from a list of available connections
+
+    When Camilla selects the "Twitter Example" connection
+    And she selects "Mention" integration action
+    Then she is prompted to select a "Finish" connection from a list of available connections
+
+    When Camilla selects the "Salesforce Example" connection
+    And she selects "Create Opportunity" integration action
+    Then she is presented with the "Add a Step" button
+    
+    When Camilla click on the "Add a Step" button
+    Then she is presented with a add step page
+    Then she selects "Log" integration step
+    And she is presented with a "Log" step configure page
+    Then she fill configure page for "Log" step with "log 1" parameter
+    And click on the "Next" button
+    
+    When Camilla clicks on the "Add a Step" button
+    Then Camilla is presented with the "Add a step" link
+    And clicks on the random "Add a step" link
+    Then she selects "Filter" integration step
```

Comment:
`Filter` is renamed to `Basic Filter` after PR on adding Filter UI was merged. Sorry about another turnaround.